app/reports/reports_list.tsx

```diff
@@ -37,7 +38,7 @@ export default function ReportsList({ userId }: ReportsListProps) {
                             <div className={`${index % 2 === 0 ? 'bg-orange-200' : 'bg-red-200'} h-full w-2 absolute left-0 top-0 rounded-s-md`}></div>
                             <div className="text-left mx-6 md:mx-8">
                                 <p className="text-sm md:text-base font-semibold">{report.test_name}</p>
-                                <p className="text-gray-700 text-sm md:text-base mt-2">Rank: {report.rank}</p>
+                                <p className="text-gray-700 text-sm md:text-base mt-2">Date attempted: {formatDate(report.start_date) ?? "Date not available"}</p>
```

Comment:
will formatDate() return null or undefined for this to work? the function seems to expect a certain format

Reply:
No it won't as long as the dateStr argument passed to it is in the expected format of "YYYY-MM-DD"(the same format `start_date` is stored.